docs/apache-airflow-providers-elasticsearch/logging/index.rst

```diff
@@ -61,7 +61,7 @@ cert, etc.) use the ``elasticsearch_configs`` setting in your ``airflow.cfg``
     remote_logging = True
 
     [elasticsearch_configs]
-    use_ssl=True
+    use_ssl=True # (This setting is valid only for ``elasticsearch<8``)
```

Comment:
Maybe we just need to remove it from the documentation?

Reply:
maybe but that depends on what I asked in https://github.com/apache/airflow/pull/33281#issuecomment-1816741321

Reply:
As far as I remember it is only related to compatible parameters names and should be fixed in: 
- https://github.com/apache/airflow/pull/34119
- https://github.com/apache/airflow/issues/34099